File: Modul-04/SOAL 3/PRAK403-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int NILAI1, NILAI2;

    printf("Input: \n");
    scanf("%d %d", &NILAI1, &NILAI2);

    printf("Output: \n");
    if (NILAI1 > NILAI2) {
        for (int i = NILAI1; i >= NILAI2; i--) {
            printf("%d %d", i, NILAI1 + NILAI2 - i);
            if (i != NILAI2) {
                printf(" - ");
            }
        }
    } else {
        for (int i = NILAI1; i <= NILAI2; i++) {
            printf("%d %d", i, NILAI1 + NILAI2 - i);
            if (i != NILAI2) {
                printf(" - ");
            }
        }
    }

    return 0;
}

File: Modul-04/SOAL 3/PRAK403-2310817210009-MuhammadIbnuSina.py
NILAI1, NILAI2 = map(int, input("Input: \n").split())

print("Output: ")
if NILAI1 > NILAI2:
    for i in range(NILAI1, NILAI2 - 1, -1):
        print(f"{i} {NILAI1 + NILAI2 - i}", end="")
        if i != NILAI2:
            print(" - ", end="")
    print()
else:
    for i in range(NILAI1, NILAI2 + 1):
        print(f"{i} {NILAI1 + NILAI2 - i}", end="")
        if i != NILAI2:
            print(" - ", end="")
